tests/e2e/vmgroup_volumeops_test.go

```diff
@@ -0,0 +1,300 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This test is going to create volume on the fresh testbed very first time.
+// After installing vmdk volume plugin/driver, volume creation should not be
+// failed very first time.
+
+// This test is going to cover the issue reported at #656
+// TODO: as of now we are running the test against photon vm it should be run
+// against various/applicable linux distros.
+
+package e2e
+
+import (
+	"os"
+	"testing"
```

Comment:
Yeah correct. @govint, as mentioned please take an example of `basic_test` or `restart_test` (currently checked in and found at /tests/e2e package) use the gocheck library for new tests.

As per one of comment (https://github.com/vmware/docker-volume-vsphere/issues/472#issuecomment-226411645) test framework has been integrated to write our automation scripts.